lib/lotus/cli.rb

```diff
@@ -80,13 +80,14 @@ def new(name = nil)
     end
 
     desc 'generate', 'generates a new action'
-    method_option :path,                desc: 'applications path',        type: :string, default: 'apps'
+    method_option :path,                desc: 'applications path',                                         type: :string,  default: 'apps'
+    method_option :skip_view,           desc: 'skip the creation of view and templates (only for action)', type: :boolean, default: false
     method_option :help, aliases: '-h', desc: 'displays the usage method'
 
     # @since 0.3.0
     # @api private
     def generate(type = nil, app_name = nil, name = nil)
-      if options[:help] || (type.nil? && app_name.nil? && name.nil?)
+      if options[:help] || (type.nil? && app_name.nil? && name.nil?) || (options[:skip_view] && type != 'action')
```

Comment:
thanks for adding the `type` check here. Come to think of it, there will be lots of generators, and I'd prefer to let `Lotus::Commands::Generate` throws exception if type is invalid and thus can be caught by CLI. For now, please remove it as it is not related to this PR. I welcome another PR if you are keen to take on type check.